web/views/tracker_view.ex

```diff
@@ -112,7 +112,7 @@ defmodule Healthlocker.TrackerView do
                   |> Map.get(:hours_slept)
       List.insert_at(list, 0, hours_slept)
     else
-      List.insert_at(list, 0, 0)
+      List.insert_at(list, 0, "null")
```

Comment:
The null values prevents a datapoint appearing on the chart for days the user didn't record data. If we were to keep it as 0, then a data point would appear at 0 on the graph and they do not want that (unless it was a 0 that a user entered). 

Also if we do not send back data for a day that a user did not enter a problem/sleep then chartjs plots the graph wrong